app/lib/shared/urls.dart

```diff
@@ -19,12 +20,16 @@ String niceUrl(String url) {
   return url;
 }
 
-String pkgPageUrl(String package, {String version, bool includeHost: false}) {
+String pkgPageUrl(String package,
+    {String version, bool includeHost: false, String hash}) {
```

Comment:
Nit: 'fragment' instead of 'hash'? Hash sounds like it's a hash of something.

Reply:
Good point, fixed.